sensor_msgs/Image queryImage
int64 matchId
string matchPath
